docs/concepts.rst

```diff
@@ -1,119 +1,108 @@
+A product doesn't just encompass the contents of a managed package or a single deployment. It also includes **unpackaged metadata**: extra bundles of Salesforce metadata that further tailor an org or complete the product.
 
-In a CumulusCI project, all unpackaged metadata is stored in subdirectories within the ``unpackaged`` directory. Unpackaged metadata plays multiple roles within the Product Delivery Model, including preparing an org for installing packages, adding more customization after the package or application is deployed, and customizing specific orgs that are used in the product's development process.
+In a CumulusCI project, all unpackaged metadata is stored in subdirectories within the ``unpackaged`` directory. Unpackaged metadata plays multiple roles, including preparing an org for installing packages, adding more customization after the package or application is deployed, and customizing specific orgs that are used in the product's development process.
 
-Learn more about managing unpackaged metadata in TODO: link Managing unpackaged configuration.
+Learn more about managing unpackaged metadata in [TODO: link Managing unpackaged configuration].
 
 Project Orgs & Services
 -----------------------
 
-Orgs and services are external, authenticated resources that each project uses. CumulusCI makes it easy to connect orgs, as well as services like a GitHub account or a MetaDeploy instance, to a single project or to use them across projects. 
+Orgs and services are external, authenticated resources that each project uses. CumulusCI makes it easy to connect orgs, as well as services like GitHub or MetaDeploy, to a single project, or to use them across many projects.
 
-Orgs
-^^^^
-
-Each project has its own set of orgs, including active scratch orgs, persistent orgs like a production or packaging org, and predefined scratch org configurations. CumulusCI securely stores org authentication information in its keychain, making it easy to access connected orgs at any time. The ``cci org list`` command shows all of the orgs connected to a project. Orgs can also be `shared across multiple projects <TODO: link to persistent orgs section>`_.
-
-Configuring orgs in CumulusCI is powerful, but comes with some complexity. To review all of the details, read the sections TODO: link "Scratch org environments" and TODO: link persistent org section.
-
-Services
-^^^^^^^^
+Each project has its own set of orgs, including active scratch orgs, persistent orgs like a production or packaging org, and predefined scratch org configurations. CumulusCI securely stores org authentication information in its keychain, making it easy to access connected orgs at any time. The ``cci org list`` command shows all of the orgs connected to a project. Orgs can also be shared across multiple projects.
```

Comment:
"Orgs can also be shared across multiple projects."

Why is this useful? How does CumulusCI assist in making this happen?

Reply:
Maybe I'm working on several different projects and I want to deploy all of them into the same persistent org in the end. I can connect that persistent org as a global org, instead of the default of being specific to one project. Then I can deploy things to it from any cci project.

It's more of an advanced use case and I don't think we need to do more than mention it.